docs/topics/loaders.rst

```diff
@@ -265,7 +265,7 @@ There are several ways to modify Item Loader context values:
       loader.context['unit'] = 'cm'
 
 2. On Item Loader instantiation (the keyword arguments of Item Loader
-   constructor are stored in the Item Loader context)::
+   ``__init__`` methodare stored in the Item Loader context)::
```

Comment:
```suggestion
   ``__init__`` method are stored in the Item Loader context)::
```

I’ve seen this issue in a few more replacements below, where the character right after ‘method’ (a space, a dot, a comma) was accidentally removed. Could you please take a look?

Reply:
🤦‍♂ of course

Reply:
@Gallaecio thanks for your patience, change pushed.